src/operations/expansions/offset.js

```diff
@@ -0,0 +1,29 @@
+const flatten = require('../../utils/flatten')
+
+const { geom2, path2 } = require('../../geometry')
+
+const offsetGeom2 = require('./offsetGeom2')
+const offsetPath2 = require('./offsetPath2')
+
+/**
+ * Create offset geometry(s) from the given object(s) using the given options (if any).
```

Comment:
I feel stupid: this is new right? how does it differ from expand ?

Reply:
path2...
- offset() does just returns offset path2
- expand() returns geom2, which is combined from offset(+) and offset(-)

geom2
- expand() === offset(), but there's support for holes now

geom3
- no offset()
- expand() is just like before... heavy ugly unions of spheres and cylinders